### ISSUE TITLE
Launch Browser Debug Not Working

### ISSUE THREAD

**Issue by lpender** [karma-runner/karma-phantomjs-launcher#92]
Trying to start Karma in debug mode.

Everything seems to work normally when starting Karma, but visiting the URL `http://localhost:9000/webkit/inspector/inspector.html?page=2` gives me nothing. Nothing at localhost:9000 at all. It seems like the server isn't actually being started.

Here's my [karma.conf.js](https://gist.github.com/lpender/f447cd8549b0d21dc691)

And here is the output

```
30 12 2015 12:14:21.346:INFO [karma]: Karma v0.13.16 server started at http://localhost:9876/
30 12 2015 12:14:21.357:INFO [launcher]: Starting browser PhantomJS
30 12 2015 12:14:21.371:INFO [phantomjs.launcher]: ACTION REQUIRED:
30 12 2015 12:14:21.371:INFO [phantomjs.launcher]:
30 12 2015 12:14:21.371:INFO [phantomjs.launcher]:   Launch browser at
30 12 2015 12:14:21.372:INFO [phantomjs.launcher]:   http://localhost:9000/webkit/inspector/inspector.html?page=2
30 12 2015 12:14:21.372:INFO [phantomjs.launcher]:
30 12 2015 12:14:21.372:INFO [phantomjs.launcher]: Waiting 15 seconds ...
30 12 2015 12:14:38.057:INFO [PhantomJS 1.9.8 (Mac OS X 0.0.0)]: Connected on socket gXgzoJdcDSGyHhF3AAAA with id 65290309
```

Thoughts?

**Comment by AgDude** [comment]
I see the same issue. If I start phantomjs directly the debugger works fine, but through karma it does not (even though karma prompts to open the browser). Here is the command I use to try phantomjs directly:
```
phantomjs --remote-debugger-port=9000 test.js --remote-debugger-autorun=yes
```

**Comment by Dignifiedquire** [comment]
Should be fixed in `0.2.3`
